perun/collect/bounds/run.py

```diff
@@ -6,19 +6,19 @@
 """
 from __future__ import annotations
 
+import click
 import os
 import shutil
 import time as systime
 
 from subprocess import SubprocessError
 from typing import Any
 
-import click
-
```

Comment:
Is there a reason for moving the import around? This ordering will violate PEP8 import guidelines (https://peps.python.org/pep-0008/#imports) and will trigger pylint [C0411](https://pylint.pycqa.org/en/v2.17.7/user_guide/messages/convention/wrong-import-order.html).